src/mlpack/methods/sparse_autoencoder/maximal_inputs_impl.hpp

```diff
@@ -32,10 +32,10 @@ inline void NormalizeColByMax(const arma::mat &input,
   output.set_size(input.n_rows, input.n_cols);
   for (arma::uword i = 0; i != input.n_cols; ++i)
   {
-    const double max = arma::max(arma::abs(input.col(i)));
-    if (max != 0.0)
+    const double max_ = max(arma::abs(input.col(i)));
+    if (max_ != 0.0)
```

Comment:
```suggestion
    const double colMax = max(arma::abs(input.col(i)));
    if (colMax != 0.0)
```
Someday I will get the style checker job working again correctly, and it will break on this (underscores in variable names).  Anyway, I get why you had to make the change, just a slightly different naming suggestion.

Reply:
Actually, that is good to know, I did not know that underscore was not part of our style.